tests/helpers/toLowerCase.test.js

```diff
@@ -1,11 +0,0 @@
-var helpers = require('../../js/main.js');
```

Comment:
Yeah me too, I'll get them back to being like that. I just need to find a way to concat them into the tests file. I don't want to have to require handlebars and chai in every single test